### ISSUE TITLE
Beach Plank Repair Tweak

### ISSUE THREAD

**Issue by JESSEQUEST** [StardewModders/mod-ideas#408]
There's no way on Earth that tiny freaking plank would take twice the amount of wood needed to build a gosh darn mill! Down with the resource costs, I say!

Make it cost one wood to build the plank. Just one. The thing isn't really luxurious looking, is it? At max that shoddy looking thing is built with 5 or maybe 10 wood.

Something I think might make it more interesting from the get go though? Make it cost 1 hardwood. It's not incredibly easy to grab early game, but if you're lucky you may get one as a drop in the cavern. And if not, it's a good reason to upgrade your axe.

TL;DR
Bridge would now cost somewhere between 1 and 10 wood to make, or 1 hardwood if you want to make it interesting to unlock.

**Comment by Annosz** [comment]
Hi! I want to get into modding, and this looks like a good place to start. I want to take it! :)

**Comment by Annosz** [comment]
The mod is now available on the [Nexus ](https://www.nexusmods.com/stardewvalley/mods/5826)and the source code can be inspected in [my repository](https://github.com/Annosz/StardewValleyModding/tree/master/StardewValleyMods/CheaperBeachBridgeRepair).

**Comment by JESSEQUEST** [comment]
Hey, thank you! I really appreciate that. Glad you could use this idea.
